csharp/test/Ice/optional/AllTests.cs

```diff
@@ -1962,6 +1963,14 @@ internal static void Populate<T>(T[] arr, T value)
                 }
             }
 
+            internal static void Populate<T>(List<T> list, T value)
```

Comment:
We can also use `new List<...>(Enumerable.Repeat(value, size));` instead of this helper.